package.json

```diff
@@ -84,6 +84,7 @@
     "fabric8-stack-analysis-ui": "0.1.40",
     "ng2-modal": "0.0.25",
     "ngx-base": "1.2.9",
+    "ng2-bootstrap": "1.3.3",
```

Comment:
@jstrachan "ng2-bootstrap" was already in the devDependency section. I'm not sure that you need it here as it should also be in the -ui too. But you should remove it from one of the sections and only have one version listed.